util/multicluster/multicluster.go

```diff
@@ -0,0 +1,17 @@
+package multicluster
+
+import (
+	"os"
+	"strconv"
+)
+
+// IsEnabled is a flag for whether the new multi-cluster feature is
+// enabled
+func IsEnabled() bool {
```

Comment:
Maybe hold off raising a PR until something more substantial? Also, look for other examples of bool flags in code base.

Reply:
My thought was to quickly set up the feature flag convention so that @isubasinghe and I could both start working on issues in parallel

The rest of the codebase does:
```
if os.Getenv("ARGO_PPROF") == "true" {
    ...
}
```
Happy to switch to that if you'd prefer